Content.Server/WebPlacer/WebPlacerSystem.cs

```diff
@@ -0,0 +1,100 @@
+using System.Linq;
+using Content.Server.Popups;
+using Content.Shared.Maps;
+using Content.Shared.WebPlacer;
+using Content.Shared.Whitelist;
+using Robust.Server.GameObjects;
+using Robust.Shared.Audio;
+using Robust.Shared.Audio.Systems;
+using Robust.Shared.Map;
+using Robust.Shared.Map.Components;
+
+namespace Content.Server.WebPlacer;
+
+/// <summary>
+/// Spawns entities (probably webs) around the component owner. Action handled by <see cref="SharedWebPlacerSystem"/>.
+/// </summary>
+public sealed class WebPlacerSystem : SharedWebPlacerSystem
+{
+    [Dependency] private readonly EntityLookupSystem _lookup = default!;
+    [Dependency] private readonly EntityWhitelistSystem _whitelistSystem = default!;
+    [Dependency] private readonly ITileDefinitionManager _tile = default!;
+    [Dependency] private readonly PopupSystem _popup = default!;
+    [Dependency] private readonly SharedAudioSystem _audio = default!;
+    [Dependency] private readonly SharedMapSystem _map = default!;
+
+    public override void Initialize()
+    {
+        base.Initialize();
+
+        SubscribeLocalEvent<WebPlacerComponent, SpiderWebActionEvent>(OnSpawnNet);
+    }
+
+    private void OnSpawnNet(EntityUid uid, WebPlacerComponent component, SpiderWebActionEvent args)
```

Comment:
```suggestion
    private void OnSpawnNet(Entity<WebPlacerComponent> webPlacer, SpiderWebActionEvent args)
```

use modern Entity<>. You can access component by `webPlacer.Comp`